test/test.js

```diff
@@ -869,88 +847,12 @@ describe('CodeService', function () {
-        .post('/vsac/ws/Ticket/TGT-INVALID-TEST', {
-          service: 'http://umlsks.nlm.nih.gov'
-        })
-        .reply(401); // Unauthorized
-
-      const vsList = [
-        {
-          name: 'Systolic Blood Pressure',
-          id: '2.16.840.1.113883.3.526.3.1032',
-          version: '20170320'
-        }
-      ];
-      return service
-        .ensureValueSetsWithAPIKey(vsList, apiKey)
-        .then(function () {
-          should.fail(0, 1, 'This code should never be executed');
-        })
-        .catch(function (error) {
-          error.should.have.length(1);
-          error[0].should.be.an('error');
-          error[0].message.should.contain('2.16.840.1.113883.3.526.3.1032');
-        });
-    });
-
-    it('should error if invalid service granting ticket is supplied', function () {
-      // Technically this should only happen if there is an issue w/ VSAC, but let's be sure we handle it
-      nock('https://vsac.nlm.nih.gov')
-        // Ticket granting ticket
-        .post('/vsac/ws/Ticket', { apikey: apiKey })
-        .reply(200, 'TGT-TEST')
-        // Service ticket (invalid)
-        .post('/vsac/ws/Ticket/TGT-TEST', {
-          service: 'http://umlsks.nlm.nih.gov'
-        })
-        .reply(200, 'ST-INVALID-TEST')
-        // Simulate response to requesting value set w/ invalid service granting ticket
-        .get('/vsac/svs/RetrieveValueSet')
-        .query({
-          id: '2.16.840.1.113883.3.526.3.1032',
-          version: '20170320',
-          ticket: 'ST-INVALID-TEST'
-        })
-        .reply(401, 'Unauthorized'); // This is the only one that puts 'Unauthorized' in the body
-
-      const vsList = [
-        {
-          name: 'Systolic Blood Pressure',
-          id: '2.16.840.1.113883.3.526.3.1032',
-          version: '20170320'
-        }
-      ];
-      return service
-        .ensureValueSetsWithAPIKey(vsList, apiKey)
-        .then(function () {
-          should.fail(0, 1, 'This code should never be executed');
-        })
-        .catch(function (error) {
-          error.should.have.length(1);
-          error[0].should.be.an('error');
-          error[0].message.should.contain('2.16.840.1.113883.3.526.3.1032');
-        });
-    });
-
     it('should error if value set is not found', function () {
       // Technically this should only happen if there is an issue w/ VSAC, but let's be sure we handle it
       nock('https://vsac.nlm.nih.gov')
-        // Ticket granting ticket
-        .post('/vsac/ws/Ticket', { apikey: apiKey })
-        .reply(200, 'TGT-TEST')
-        // Service ticket and VS retrieval #1
-        .post('/vsac/ws/Ticket/TGT-TEST', {
-          service: 'http://umlsks.nlm.nih.gov'
-        })
-        .reply(200, 'ST-TEST')
+        // VS retrieval #1
         .get('/vsac/svs/RetrieveValueSet')
+        .basicAuth({ user: 'apikey', pass: apiKey })
         .query({
           id: '1.2.3.4.5.6.7.8.9.10',
           version: '20170320',
```

Comment:
I'm one line off here, but the next line still has `ticket: 'ST-TEST'`, which I don't think we need in the query anymore.

Reply:
Fixed in [e0f6fda](https://github.com/cqframework/cql-exec-vsac/pull/27/commits/e0f6fdaee26cd4e0456388962c376da585bee218).